test/spec/smartbanner_spec.js

```diff
@@ -110,6 +110,74 @@ describe('SmartBanner', function() {
 
     });
 
+    context('direct api usage', function() {
+      const HTML_WITH_API = `<!doctype html>
+        <html style="margin-top:10px;">
+        <head>
+          <meta charset="utf-8">
+          <meta name="smartbanner:api" content="true">
+        </head>
+        <body>
+        </body>
+      </html>`;
+
+      context('when we append meta api and no arguments in constructor', function() {
```

Comment:
Let's change to _when API enabled, but no constructor parameters_ to be clearer.